civis/io/_tables.py

```diff
@@ -1029,8 +1029,16 @@ def civis_file_to_table(file_id, database, table, client=None,
 
     # Use Preprocess endpoint to get the table columns as needed
     # and perform necessary file cleaning
-    need_table_columns = ((not table_exists or existing_table_rows == 'drop')
+    table_columns_have_sql = True
```

Comment:
nit: this seems a bit wordy maybe something like `table_columns_valid`, idk, thoughts?

Reply:
Yeah, that seems better. I wanted to be specific, but also thought it sounded awkward as is.